components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/IdentityManagementConfigImpl.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.handler.event.account.lock;
+
+import org.apache.commons.lang.StringUtils;
+import org.wso2.carbon.identity.base.IdentityConstants;
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+import org.wso2.carbon.identity.governance.IdentityGovernanceException;
+import org.wso2.carbon.identity.governance.common.IdentityConnectorConfig;
+import org.wso2.carbon.identity.handler.event.account.lock.constants.IdentityManagementEndpointConstants;
+import org.wso2.carbon.utils.multitenancy.MultitenantConstants;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+public class IdentityManagementConfigImpl implements IdentityConnectorConfig {
+
+        private static String connectorName = "identity-management";
+
+        @Override
+        public String getName() {
+            return connectorName;
+        }
+
+        public String getFriendlyName() {
+            return "Identity Management Endpoints";
+        }
+
+        @Override
+        public String getCategory() {
+            return "DEFAULT";
+        }
+
+        @Override
+        public String getSubCategory() {
+            return "DEFAULT";
+        }
+
+        @Override
+        public int getOrder() {
+            return 0;
+        }
+
+        public Map<String, String> getPropertyNameMapping() {
+            Map<String, String> nameMapping = new HashMap<>();
+            nameMapping.put(IdentityManagementEndpointConstants.RECOVERY_REST_ENDPOINT, "Recovery Rest Endpoint");
+            nameMapping.put(IdentityManagementEndpointConstants.SELF_REGISTRATION_ENDPOINT, "Self Registration Endpoint");
+            return nameMapping;
+        }
+
+        @Override
+        public Map<String, String> getPropertyDescriptionMapping() {
+            Map<String,String> descriptionMap = new HashMap<>();
+            descriptionMap.put(IdentityManagementEndpointConstants.RECOVERY_REST_ENDPOINT, "endpoint");
```

Comment:
why are we adding "endpoint" as the description?
Shouldn't we add a meaningful description for each endpoint